erigon-lib/commitment/hex_patricia_hashed.go

```diff
@@ -2004,7 +2007,15 @@ func (hph *HexPatriciaHashed) Process(ctx context.Context, updates *Updates, log
 		start        = time.Now()
 		logEvery     = time.NewTicker(20 * time.Second)
 	)
-	defer logEvery.Stop()
+	CurrentCommitmentMetrics = CommitmentMetrics{}
+	CurrentCommitmentMetrics.Updates = int(updates.Size())
+	defer func() {
+		logEvery.Stop()
+		CurrentCommitmentMetrics.TotalFoldingTime /= 100   // to milliseconds
```

Comment:
oh god no please. use `time.Duration` directly. so it will be just 
`CurrentCommitmentMetrics.TotalFoldingTime += time.Since(start)`

Reply:
and no sense to cast to ms straight away.